app/src/main/java/com/cornellappdev/android/eatery/Model/MealModel.java

```diff
@@ -1,61 +1,49 @@
 package com.cornellappdev.android.eatery.Model;
 
+import android.support.annotation.NonNull;
+import org.threeten.bp.LocalDateTime;
+
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.HashMap;
+import java.util.List;
 
 /**
- * Created by abdullahislam on 2/18/18.
+ * Model class to represent each meal
  */
 
-public class MealModel implements Serializable{
-    private Date start;
-    private Date end;
-    private String type;
-    private HashMap<String, ArrayList<String>> menu;
-
-    // toString() but better
-    public String stringTo() {
-        String info = type + " on " + " from: " + start.toString() + " to: " + end.toString() + "\n";
-        for (HashMap.Entry<String, ArrayList<String>> entry : menu.entrySet()) {
-            String key = entry.getKey();
-            Object value = entry.getValue();
-            info = info + " " + key + ": " + value.toString() + "\n";
-        }
-        return info;
-    }
-
-    public Date getStart() {
-        return start;
-    }
+public class MealModel extends Interval implements Serializable {
+    private MealMenuModel menu;
+    private MealType type;
 
-    public void setStart(Date start) {
-        this.start = start;
+    MealModel(@NonNull LocalDateTime start, @NonNull LocalDateTime end) {
+        super(start, end);
     }
 
-    public Date getEnd() {
-        return end;
-    }
-
-    public void setEnd(Date end) {
-        this.end = end;
+    //TODO: Keep it for now for testing. Get rid of it later
```

Comment:
Get rid of TODOs from code base, simply leave comment on why it is there and create an issue if you want to make todos. Also there should be a space after the //

Reply:
Also please change the name of this method from stringTo to avoid confusion